instrumentation/aws-java-sdk-kinesis-2.1.0/src/main/java/software/amazon/awssdk/core/client/handler/AsyncClientHandler_Instrumentation.java

```diff
@@ -11,18 +11,14 @@
 import java.util.concurrent.CompletableFuture;
 
 @Weave(originalName = "software.amazon.awssdk.core.client.handler.AsyncClientHandler", type = MatchType.Interface)
-public class AsyncClientHandler_Instrumentation {
-    // This prevents further traces from forming when using the async client
-    @Trace(leaf = true, excludeFromTransactionTrace = true)
-    public <InputT extends SdkRequest, OutputT extends SdkResponse> CompletableFuture<OutputT> execute(
-            ClientExecutionParams<InputT, OutputT> executionParams) {
-        return Weaver.callOriginal();
-    }
+public abstract class AsyncClientHandler_Instrumentation {
```

Comment:
These methods simply called `Weaver.callOriginal()`. Removed the call and made the methods/class abstract.